packages/mdc-switch/_variables.scss

```diff
@@ -1,33 +1,33 @@
-// 
+//
 // Copyright 2016 Google Inc. All Rights Reserved.
-// 
+//
 // Licensed under the Apache License, Version 2.0 (the "License");
 // you may not use this file except in compliance with the License.
 // You may obtain a copy of the License at
-// 
+//
 //      http://www.apache.org/licenses/LICENSE-2.0
-// 
+//
 // Unless required by applicable law or agreed to in writing, software
 // distributed under the License is distributed on an "AS IS" BASIS,
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-// 
+//
 
 $mdc-switch-track-width: 34px;
 $mdc-switch-track-height: 14px;
 $mdc-switch-knob-diameter: 20px;
 $mdc-switch-focus-ring-diameter: 48px;
 $mdc-switch-knob-active-margin: $mdc-switch-track-width - $mdc-switch-knob-diameter;
 
-$mdc-switch-unchecked-track-color: #000;
-$mdc-switch-unchecked-knob-color: #fafafa;
-$mdc-switch-unchecked-focus-ring-color: #9e9e9e;
-$mdc-switch-unchecked-knob-color-dark: #bdbdbd;
-$mdc-switch-unchecked-track-color-dark: #fff;
-$mdc-switch-unchecked-focus-ring-color-dark: #f1f1f1;
-$mdc-switch-disabled-knob-color: #bdbdbd;
-$mdc-switch-disabled-knob-color-dark: #424242;
+$mdc-switch-unchecked-track-color: #000 !default;
```

Comment:
I am afraid that we might not accept making this a default color as we have switch unchecked and disabled color specified by [material guideline](https://material.io/guidelines/components/selection-controls.html#selection-controls-switch), for switch on however, it would by default use primary color from theme.

If you want to customize switch on color, you could do either:
```
// restrict the scope selector
.my-awesome-switch {
  --mdc-theme-primary: red;
}
<div class="mdc-switch my-awesome-switch">
     ...
</div>
```
or
```
// global primary
:root {  
  --mdc-theme-primary: red;
}
```